src/main/java/build/buildfarm/cas/cfc/CASFileCache.java

```diff
@@ -314,6 +320,14 @@ public CASFileCache(
     this.delegateSkipLoad = delegateSkipLoad;
     this.directoriesIndexDbName = directoriesIndexDbName;
 
+    if (configs.getWorker().getStorages().get(0).isPublishTtlMetric()) {
+      casTtl =
+          Histogram.build()
+              .name("cas_ttl")
+              .help("The amount of time CAS entries live in L1 storage")
```

Comment:
Can we clarify the units of this metric here?

Reply:
Fixed.  I put the units in the metric name as well (`cas_ttl_ms`).  I think a lot of our time metrics are missing units-- so that's something we might want to fix also.